s3backgrounddelete/s3backgrounddelete/object_recovery_msgbus.py

```diff
@@ -245,3 +245,24 @@ def purge(self):
             self._logger.error("Exception:{}".format(exception))
             self.__isproducersetupcomplete = False
             return False
+            
+    def get_count(self):
+        """Get count of unread messages."""
+        try:
+            consumer_group = self._config.get_msgbus_consumer_group()
+            if not self.__isproducersetupcomplete:
+                self.__setup_producer()
+                if not self.__isproducersetupcomplete:
+                    self._logger.info("producer connection issues")
+                    return 0
+            
+            value = self.__msgbuslib.count(consumer_group)
+            if value is None:
+                return 0
+            elif value > 0:
+                return value
+            else:
+                self._logger.debug("Failed to receive count of unread msgs from message bus : " + str(value))
```

Comment:
nothing is returned in this else case.
should be if value is none return 0 else return value.

Reply:
done